common/utils.py

```diff
@@ -36,4 +36,5 @@ def vmaas_post_request(endpoint, data_json, session=None):
         except requests.exceptions.RequestException:
             VMAAS_CNX_ERR.inc()
             LOGGER.exception("Error calling VMAAS: ")
+            raise
```

Comment:
we can't raise here because in evaluator and vmaas_sync services the vmaas_post_request doesn't expect any exception and would crash the application